### DISCUSSION TITLE
Dbeaver 22.1.3 feature

### DISCUSSION THREAD

**Discussion by VistaArtista** [dbeaver/dbeaver#17177] [Q&A]
Hi guys!
I'm using dbeaver at work and recently have updated to 22.1.3
Now data shows me like this (like [Object] instead of json data inside):
![image](https://user-images.githubusercontent.com/80551087/180997456-012a473f-5a40-4f1c-be6e-6446c899d58b.png)

How to fix this?? 
Thanks in advance!

**Comment by LonwoLonwo** [comment]
Hello @VistaArtista 

What database do you use?
Can you provide your table DDL and data sample for it?

**Reply by VistaArtista** [reply]
I'm using Hive. It is not possible to add data sample
columns with [Object] - is struct with arrays and structs inside.
Thing is, in previous versions of Dbeaver i was able to see all data inside

**Reply by LonwoLonwo** [reply]
I can't help without more information about table DDL and any info about data inside.

Also, did you try to check the value panel on your cells?

![2022-07-28 00_30_45-DBeaver Enterprise 22 1 3 - xml_data](https://user-images.githubusercontent.com/45152336/181376124-08dd6fff-259c-4c74-8f0c-6acf4aeea132.png)
